### PR TITLE
Create script for running migrations

### PR DESCRIPTION
**Pull request by annalach**
- created script
- created SQL files (two, to start)
- created docker-compose.yml file
- updated README.md
- created Decision Log

### CODE REVIEW

File: run_migrations.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import asyncpg
+
+from settings import BASE_PATH, CONNECTION_STRING
+
+
+async def main():
+    MIGRATIONS_DIR = os.path.join(BASE_PATH, 'migrations')
+
+    scripts = sorted(os.listdir(MIGRATIONS_DIR))
+    always_run_script, migration_scripts = scripts[0], scripts[1:]
```
Comment (Mateusz-Slisz) [inline run_migrations.py:13]:
```suggestion
    always_run_script, *migration_scripts = scripts
```

File: run_migrations.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import asyncio
+import os
+
+import asyncpg
+
+from settings import BASE_PATH, CONNECTION_STRING
+
+
+async def main():
+    MIGRATIONS_DIR = os.path.join(BASE_PATH, 'migrations')
```
Comment (Mateusz-Slisz) [inline run_migrations.py:10]: Maybe it would be cleaner to add that variable to settings

File: run_migrations.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    connection = await asyncpg.connect(CONNECTION_STRING)
+
+    async with connection.transaction():
+        always_run_script_path = os.path.join(MIGRATIONS_DIR,
+                                              always_run_script)
+        with open(always_run_script_path) as f:
+            sql_statement = f.read()
+            await connection.execute(sql_statement)
+            print(f'Executed {always_run_script}')
```
Comment (wysockipiotr) [inline run_migrations.py:23]: Here it doesn't matter so much, but we have to remember that reading from the disk blocks the event loop. Consistent use of [aiofiles](https://github.com/Tinche/aiofiles) could be considered. 

```python
async with aiofiles.open(always_run_script_path) as f:
    sql_statement = await f.read()
    await connection.execute(sql_statement)
    print(f'Executed {always_run_script}')
```

File: run_migrations.py (hunk 1 above)
Comment (annalach) [reply run_migrations.py]: Cool! thanks, updated 🙂

File: run_migrations.py (hunk 2 above)
Comment (annalach) [reply run_migrations.py]: It makes sense to keep this variable in settings with others, even if it's used only here. Moved it 😉

File: run_migrations.py (hunk 3 above)
Comment (annalach) [reply run_migrations.py:21]: I agree, we have to use nonblocking solution when reading SQL files in models

**Pull request merged**